.env.local.example

```
@@ -1,65 +1,44 @@
-OPENROUTER_API_KEY=
+# Convex deployment configuration
+CONVEX_DEPLOYMENT=
+NEXT_PUBLIC_CONVEX_URL=
 
-# Web Search API Key (Optional - enables web search functionality)
-# Get your API key at: https://exa.ai/
-# EXA_API_KEY=
+# WorkOS Authentication (Required for user management and conversation persistence)
+WORKOS_API_KEY=sk_example_123456789
+WORKOS_CLIENT_ID=client_123456789
+WORKOS_COOKIE_PASSWORD= # generate a secure password (min 32 characters)
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)